charms/istio-gateway/tests/test_charm.py

```diff
@@ -21,95 +15,70 @@ def test_no_kind(harness):
     assert harness.charm.model.unit.status == BlockedStatus('Config item `kind` must be set')
 
 
-def test_kind_no_rel(harness, mocker):
+def test_kind_no_rel(harness, helpers):
     harness.set_leader(True)
-
     harness.update_config({'kind': 'ingress'})
-    harness.begin_with_initial_hooks()
-
-    container = harness.model.unit.get_container('noop')
-    harness.charm.on['noop'].pebble_ready.emit(container)
+    helpers.begin_noop(harness)
 
     assert harness.charm.model.unit.status == BlockedStatus('Waiting for istio-pilot relation')
 
 
-def test_kind_ingress(harness, mocker):
```

Comment:
are these deleted tests no longer valid?

Reply:
the deleted tests were combined into one paramterized test with the kind feature. The egress/ingress tests were the same test, with the difference being one used the ingress example file and the other used the egress file.